### PR TITLE
SmartCardContext.getStatusChange() algorithm

### PR DESCRIPTION
**Pull request by dandrader**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/web-smart-card/pull/14.html" title="Last updated on Jul 26, 2023, 12:09 PM UTC (8e783e7)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/web-smart-card/14/c82b460...8e783e7.html" title="Last updated on Jul 26, 2023, 12:09 PM UTC (8e783e7)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -219,15 +230,124 @@ <h3><dfn>listReaders()</dfn> method</h3>
...
+          </ol>
+
+          <p>The <dfn>high word</dfn> of a [[PCSC5]] `DWORD` is the result of an
+          unsigned right shift of 16 bits on that `DWORD`.</p>
+
+          <p>To <dfn>set the high word</dfn> of a [[PCSC5]] `DWORD` named
+          |dword:DWORD| to a given number |n|, perform the following steps.</p>
+          <ol>
+            <li>Let |shiftedN| be the result of a left shift of 16 bits on |n|.</li>
+            <li>Set |dword| to the bitwise OR of |dword| and |shiftedN|.</li>
```
Comment (reillyeon) [inline index.html:338]: Clear the high bits of `dword` first?

File: index.html [hunk 2]
```diff
@@ -298,6 +443,47 @@ <h5><dfn>SmartCardReaderStateFlagsIn</dfn> dictionary</h4>
               <dd>The application believes that there is an unresponsive card in
                 the reader.</dd>
             </dl>
+
+            <p>The [[PCSC5]] `DWORD`
+              <dfn data-dfn-for=SmartCardReaderStateFlagsIn>corresponding</dfn>
+              to a given {{SmartCardReaderStateFlagsIn}} is created with the
+              following steps:</p>
+            <ol>
+              <li>Let |flagsIn:SmartCardReaderStateFlagsIn| be that given
```
Comment (reillyeon) [inline index.html:452]:
```suggestion
              <li>Let |flagsIn:SmartCardReaderStateFlagsIn| be the given
```

File: index.html [hunk 3]
```diff
@@ -298,6 +443,47 @@ <h5><dfn>SmartCardReaderStateFlagsIn</dfn> dictionary</h4>
...
+                |pcscFlags|.</li>
+              <li>If |flagsIn|.["{{SmartCardReaderStateFlagsIn/inuse}}"] is
+                `true`, [=add a flag|add=] [[PCSC5]] `SCARD_STATE_INUSE` to
+                |pcscFlags|.</li>
+              <li>If |flagsIn|.["{{SmartCardReaderStateFlagsIn/mute}}"] is
+                `true`, [=add a flag|add=] `SCARD_STATE_MUTE` to
+                |pcscFlags|.
+                <aside class="note">
+                  <p>`SCARD_STATE_MUTE` is not part of [[PCSC5]] but it is still
+                  considered in this speficiation as existing PC/SC
```
Comment (reillyeon) [inline index.html:481]:
```suggestion
                  considered in this specification as existing PC/SC
```

File: index.html [hunk 4]
```diff
@@ -329,6 +515,52 @@ <h4><dfn>SmartCardReaderStateOut</dfn> dictionary</h4>
...
+                  [=SmartCardReaderStateFlagsOut/corresponding=]
+                  to |pcscState|.`EventState`.</li>
+                <li>Set |stateOut|.["{{SmartCardReaderStateOut/eventCount}}"] to
+                  the [=SmartCardContext/high word=] of |pcscState|.`EventState`.
+                  <aside class="note" title="Number of card insertion and removal
+                    events">
+                    <p>Having the number of card insertion and removal events
+                    stored in the upper 2 bytes of the `EventState` and
+                    `CurrentState` `DWORD`s is not part of [[PCSC5]] but it is
+                    still considered in this speficiation as several existing
```
Comment (reillyeon) [inline index.html:543]:
```suggestion
                    still considered in this specification as several existing
```

File: index.html [hunk 5]
```diff
@@ -329,6 +515,52 @@ <h4><dfn>SmartCardReaderStateOut</dfn> dictionary</h4>
...
+                    PC/SC implementations use it.</p>
+                    <p>In PC/SC implementations where this information is not
+                    avaialble, the upper two bytes of `EventState` will always
```
Comment (reillyeon) [inline index.html:546]:
```suggestion
                    available, the upper two bytes of `EventState` will always
```

File: index.html [hunk 6]
```diff
@@ -329,6 +515,52 @@ <h4><dfn>SmartCardReaderStateOut</dfn> dictionary</h4>
...
+                    be zero and so the resulting
+                    "{{SmartCardReaderStateOut/eventCount}}".</p>
```
Comment (reillyeon) [inline index.html:548]:
```suggestion
                    "{{SmartCardReaderStateOut/eventCount}}" will be zero.</p>
```

**Review by reillyeon — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (dandrader) [reply index.html]: Oops! Fixed, thanks!

**Review by reillyeon — APPROVED** [review]

**Pull request merged**